src/examples/install_latest_version.yml

```diff
@@ -0,0 +1,22 @@
+description: >
+  Install a specific version of Keeper Secrets Manager CLI.
+usage:
+  version: 2.1
+
+  orbs:
+    keeper: jgiovaresco/keeper@x.y.z
```

Comment:
is this an oversight? or the orb is still under `jgiovaresco/keeper`?

Reply:
🙈